bin/start-agent.sh

```diff
@@ -22,6 +23,7 @@ pushd $rootdir/src/integration-test/resources/agent
 echo "[AmazonCloudWatchAgent]
 aws_access_key_id = $AWS_ACCESS_KEY_ID
 aws_secret_access_key = $AWS_SECRET_ACCESS_KEY
+aws_session_token = $AWS_SESSION_TOKEN
```

Comment:
Can you try running integration tests with an IAM user with no session token variable set, and see if this still works?

Also, we'll want this on master later as well.

Reply:
It is working with IAM user. Does it mean we will always keep a copy of access key pair or create new access key pair (if the secret access key isnt stored locally because it cannot be viewed in the console after creation).

Reply:
Okay cool. We still want to be using temporary credentials with a session token when possible, I just want to make sure it will still continue to work with long-lived credentials (if someone running the tests chooses to do that).